processor/src/main/java/org/mobilitydata/gtfsvalidator/processor/EnumGenerator.java

```diff
@@ -42,6 +43,15 @@ public static String createEnumName(String interfaceName) {
         return interfaceName.substring(0, interfaceName.length() - ENUM_SUFFIX.length());
     }
 
+    private static MethodSpec getNumberMethod() {
```

Comment:
this piece of code was just moved around?

Reply:
No, it became static. It was moved by IDEA automatically because it rearranges static methods.

Reply:
oh, alright. Thks for the details!